docs/rrtmg_clouds.rst

```diff
@@ -0,0 +1,220 @@
+=======================
+RRMTG Clouds with McICA
+=======================
+
+A brief description of the different options that can be used in RRTMG with McICA and the input parameters required in each case.
+
+Cloud properties
+================
+
+There are three options for the RRTMG ``inflag``, as given in the ``climt`` dictionary: rrtmg_cloud_props_dict_
+
+.. _rrtmg_cloud_props_dict: https://github.com/CliMT/climt/blob/39c1bcacd3b348ec63000d4b57d525e523203883/climt/_components/rrtmg/rrtmg_common.py#L18-L22
+
+.. code-block:: python
```

Comment:
There needs to be a blank line between the code-block line and the code itself.

Reply:
The same for all instances.

Reply:
Thanks!